packages/http/README.md

```diff
@@ -1,3 +1,214 @@
 # Prism HTTP
 
-TBD
+This package provides HTTP APIs for forwarding and mocking requests.
```

Comment:
The HTTP client package is meant to be a generic HTTP client which has the ability to also mock responses. We should be advertising it like that, and we want to avoid talking about forwarding because that’s an internal implementation detail of a proxy we no longer use.